lib/server/connect.js

```diff
@@ -87,12 +163,14 @@ Meteor.publish('kadira.debug.timeline', function() {
 
 
 Meteor.publish('kadira.debug.listeners', function() {
+  var kadiraInfo = Kadira._getInfo();
+  var sessionId = kadiraInfo.session;
+
```

Comment:
You need to authorize here as well. May be it's better to move the auth code to a common place.